src/main/java/com/jcabi/github/mock/MkRepo.java

```diff
@@ -281,7 +281,10 @@ public Stars stars() {
 
     @Override
     public Notifications notifications() {
-        return new MkNotifications(0);
+        return new MkNotifications(
+            this.storage,
+            this.xpath().concat("/notifications/notification")
```

Comment:
@llorllale so basically, in the XML tree, notifications will be an array of child elements under ``<notifications>``,  like this:

```xml
<notificatons>
    <notification>
        <id></id>
    </notification>
</notifications>
```

right?

Reply:
@amihaiemil yes, I've updated the PR's description